### PR TITLE
Enable the observability GitHub topic

### PR DESCRIPTION
**Pull request by v1v**
### What

Enable the observability topic

### CODE REVIEW

File: resources/label-definitions.properties [hunk 1]
```diff
@@ -685,6 +685,7 @@ openshift-pipeline=pipeline scm-related post-build builder misc
 openshift-sync=cluster builder scm-related external
 openstack-cloud=cluster slaves
 openstack-heat=builder
+opentelemetry=observability administrative-monitor monitoring
```
Comment (oleg-nenashev) [inline resources/label-definitions.properties:688]: Better to use GitHub topics

Comment (daniel-beck) [reply resources/label-definitions.properties:688]: Agree, please use GH topics.

```suggestion
```

Comment (v1v) [reply resources/label-definitions.properties:688]: I thought it was required both! Thanks for the heads up :)

**Review by daniel-beck — APPROVED** [review]
Thanks! I'll merge this later today after I'm done with the security advisory.

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**